### PR TITLE
Redirection to list after adding new picture

### PR DESCRIPTION
**Pull request by vllorca**

### CODE REVIEW

File: src/modules/RA/Picture/views/Create.js [hunk 1]
```diff
@@ -4,13 +4,16 @@ import { Create } from 'react-admin';
 import PictureFields from '../components/PictureFields';
 import DefaultActions from '../../../../components/react-admin/DefaultActions';
 
+const redirect = (basePath, id, data) => `/viewpoint/${data.viewpoint}`;
+
 export const PictureCreate = props => (
   <Create
     {...props}
-    actions={<DefaultActions />}
+    actions={<DefaultActions redirect={redirect} />}
```
Comment (jrmi) [inline src/modules/RA/Picture/views/Create.js:12]: Does the redirection works if we don't came from a viewpoints ?

Comment (vllorca) [reply src/modules/RA/Picture/views/Create.js:12]: Yes it does. I changed the redirection in the Create component
It's the one called when you add a picture, coming from a viewpoint or directly from the picture list.

Comment (jrmi) [reply src/modules/RA/Picture/views/Create.js:12]: Actually, we want to come back to the viewpoint if we came from it but come back to picture list if we don't. Is this the implemented behaviour ?

Comment (vllorca) [reply src/modules/RA/Picture/views/Create.js:12]: As describe in the ticket#791, after adding a picture we want to come back to the list of photo from the corresponding viewpoint, and not the pictures list for all the observatory.

image

Comment (jrmi) [reply src/modules/RA/Picture/views/Create.js:12]: Yes but only when you came from a viewpoint as the first words explain it.

**Review by jrmi — APPROVED** [review]

**Review by CFarcy — APPROVED** [review]

**Pull request merged**